admin/scripts/modules/aktivity/_Import/ImportValuesSanitizer.php

```diff
@@ -912,7 +914,20 @@ private function getValidatedStart(array $activityValues, ?Aktivita $originalAct
                 )]
             );
         }
-        return $this->createDateTimeFromRangeBorder($activityValues[ExportAktivitSloupce::DEN], $activityValues[ExportAktivitSloupce::ZACATEK], 'začátek');
+        $startDateTime = $this->createDateTimeFromRangeBorder($activityValues[ExportAktivitSloupce::DEN], $activityValues[ExportAktivitSloupce::ZACATEK], 'začátek');
+        if ($startDateTime->isSuccess()) {
+            $startHour = (int)(DateTimeCz::createFromFormat(DateTimeCz::FORMAT_DB, $startDateTime->getSuccess())->format('G'));
+            if (($startHour < PROGRAM_ZACATEK) && $startHour >= PROGRAM_KONEC) {
+                return ImportStepResult::successWithErrorLikeWarnings(
+                    null,
+                    [sprintf(
+                        'Aktivita má uvedený začátek (%s), který je mimo rozsah denního začátku a konce GameConu.',
```

Comment:
Přidal bych do hlášky rovnou i ten správný začátek a konec "dne", PROGRAM_ZACATEK a PROGRAM_KONEC, něco jako
```suggestion
                        'Aktivita má uvedený začátek (%s), který je mimo rozsah denního začátku a konce GameConu (%s - %s).',
```